HaxeManual/02-types.tex

```diff
@@ -319,6 +319,14 @@ \subsection{Inheritance}
 
 A class may override \tref{methods}{class-field-method} of its parent class, which requires the explicit \expr{override} keyword. The effects and restrictions of this are detailed in \Fullref{class-field-overriding}.
 
+\since{4.0.0}
+
+Classes can be declared with the keyword \expr{final} to prevent them from being extended.
+
+In some situations the user may still want to extend a class marked as final. This is possible by using the \expr{:hack} metadata on the \emph{extending} class. This should be avoided in general, hence the discouraging metadata name \expr{:hack}.
```

Comment:
I'd honestly not mention this at all...

Reply:
I guess it's already listed in the metadata list, but that's a bit hard to navigate … I'll remove it.

Reply:
Not sure that even works everywhere, if the generator writes a native `final` flag.